app/assets/stylesheets/views/_taxons.scss

```diff
@@ -23,13 +21,20 @@
 
   h3 {
     @include bold-19;
-    padding-bottom: $gutter-one-third;
+    @include media (tablet) {
+      padding-bottom: $gutter-one-third;
+    }
   }
 
   p {
     @include core-19;
-    margin-top: 2px;
-    margin-bottom: 6px;
+    margin-top: #{(5 / 16)}em;
+    margin-bottom: #{(20 / 16)}em;
+
+    @include media(tablet) {
+      margin-top: #{(5 / 19)}em;
+      margin-bottom: #{(20 / 19)}em;
```

Comment:
@alecgibson it's been suggested by other frontend devs on slack that we use the `$gutter` variables for this.

`margin-top: $gutter-two-thirds;`
`margin-bottom: $gutter-one-third / 2;`